app/src/forms/form/exportService.js

```diff
@@ -42,15 +42,49 @@ const EXPORT_FORMATS = Object.freeze({
 
 const service = {
 
+  _readSchemaFields: async (schema) => {
+    const fields = [];
+
+    // function to recursively find all input field names
+    const findFields = (obj) => {
+      // if component is an input field showing in the form,
+      if (obj.input && !obj.hidden) {
+        fields.push(obj.key);
+
+        // if object is a checkbox group
+        if (obj.type && obj.type.includes('checkbox') && obj.values) {
+          // add checkbox fields now
+          obj.values.forEach(element => fields.push(obj.key + '.' + element.value));
+        }
+      }
+
+      // else not an input (eg: panel, column, etc)
+      else {
+        // for each property of the component
+        Object.keys(obj).forEach(function (prop) {
```

Comment:
Use arrow notation for anonymous functions:
```js
Object.keys(obj).forEach((prop) => {
```

Reply:
can you explain your reasoning for that advice?